server/src/main/java/org/opensearch/ingest/IngestService.java

```diff
@@ -1271,6 +1271,14 @@ private static IngestDocumentWrapper toIngestDocumentWrapper(int slot, IndexRequ
         return new IngestDocumentWrapper(slot, toIngestDocument(indexRequest), null);
     }
 
+    private static List<IngestDocumentWrapper> toIngestDocumentWrappers(List<Integer> slots, List<IndexRequest> indexRequests) {
```

Comment:
I don't think this needs to be static?

Reply:
Personally I have no problem making methods static if they don't operate on any instance state. Any reason to prefer a non-static method?